<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PatriciaSQL</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>graphics/sql_512_white_bg_2.png</normaloff>graphics/sql_512_white_bg_2.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="SQLEditor" name="sqlEditorArea">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>200</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>DejaVu Sans Mono</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="cursor" stdset="0">
            <cursorShape>IBeamCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
          <widget class="PgTreeView" name="treeView">
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>200</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menusomething">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSettings"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menusomething"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionExecute"/>
   <addaction name="actionExecute_selected"/>
   <addaction name="actionExplain"/>
   <addaction name="actionExplain_Selected"/>
   <addaction name="actionSnippets"/>
   <addaction name="actionExportCSV"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About PatriciaSQL</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save...</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>--</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="icon">
    <iconset>
     <normaloff>graphics/play-circle-regular.svg</normaloff>graphics/play-circle-regular.svg</iconset>
   </property>
   <property name="text">
    <string>Execute</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>S&amp;ettings</string>
   </property>
  </action>
  <action name="actionExecute_selected">
   <property name="icon">
    <iconset>
     <normaloff>graphics/play-circle-solid.svg</normaloff>graphics/play-circle-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Execute Selected</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Return</string>
   </property>
  </action>
  <action name="actionExplain">
   <property name="icon">
    <iconset>
     <normaloff>graphics/question-circle-regular.svg</normaloff>graphics/question-circle-regular.svg</iconset>
   </property>
   <property name="text">
    <string>Explain</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExplain_Selected">
   <property name="icon">
    <iconset>
     <normaloff>graphics/question-circle-solid.svg</normaloff>graphics/question-circle-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Explain Selected</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="actionSnippets">
   <property name="icon">
    <iconset>
     <normaloff>graphics/code-solid.svg</normaloff>graphics/code-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Snippets</string>
   </property>
  </action>
  <action name="actionExportCSV">
   <property name="icon">
    <iconset>
     <normaloff>graphics/file-csv-solid.svg</normaloff>graphics/file-csv-solid.svg</iconset>
   </property>
   <property name="text">
    <string>Export to CSV</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SQLEditor</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">sqleditor.h</header>
  </customwidget>
  <customwidget>
   <class>PgTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">pgtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>showMessage()</slot>
 </slots>
</ui>
